### PR TITLE
Amc bldc: the new supervisor multi-packets

### PR DESCRIPTION
**Pull request by valegagge**
Here there is the generated code with [this model](https://github.com/icub-tech-iit/study-mbd-amc-bldc/pull/213).

The amc-bldc firmware version is `1.0.7`.

The firmware has been compiled with the icub-firmware shared at head of devel branch. (the SHA is 79cf7d52fa7e3605e510cb520b1c3b127f2c1be9).

We tested the firmware on the wrist setup and it works properly.

We added the [histogram print function](https://github.com/GrmanRodriguez/icub-firmware/blob/f0986cd5b721a47000ebb0512ab882b8ef1ebfcb/emBODY/eBcode/arch-arm/board/amcbldc/application07/src/embot_app_application_theMBDagent.cpp#L487-L514) for debugging: we developed it to check the size of input or output CAN queue. 

cc @GrmanRodriguez

### CODE REVIEW

File: emBODY/eBcode/arch-arm/board/amcbldc/application07/proj/amcbldc-application07.uvoptx [hunk 1]
```diff
@@ -148,14 +153,82 @@
           <Name>UL2CM3(-S0 -C0 -P0 -FD20000000 -FC1000 -FN1 -FF0STM32G47x-8x_512 -FS08000000 -FL080000 -FP0($$Device:STM32G474QETx$CMSIS\Flash\STM32G47x-8x_512.FLM))</Name>
         </SetRegEntry>
       </TargetDriverDllRegistry>
-      <Breakpoint/>
+      <Breakpoint>
```
[the remark this answers is not in the document]
Comment (valegagge) [reply emBODY/eBcode/arch-arm/board/amcbldc/application07/proj/amcbldc-application07.uvoptx:156]: Yes, it is not necessary.

**Review by marcoaccame — APPROVED** [review]
Thanks @valegagge and @GrmanRodriguez, the PR is OK.
I would wait the tag of the new release 2022.05 before processing the merge. I think it is a matter of hours.

**Comment by mfussi66** [comment]
Due to my last commit, lots of autogen files have their date and model version in my latest commit at earlier values compared to the overwritten ones. 
I think this is just caused by matlab being "smart" and generating only the files which have differences at the model level. 
Nonetheless, You could clearly see the latest contributions to the architectural model. I think it can be safely merged on my end.

**Comment by marcoaccame** [comment]
> Nonetheless, You could clearly see the latest contributions to the architectural model. I think it can be safely merged on my end.

Ok, so we shall merge then.

**Pull request merged**